test/USTB.t.sol

```diff
@@ -2,6 +2,7 @@ pragma solidity ^0.8.20;
 
 import "forge-std/StdUtils.sol";
 import {Test} from "forge-std/Test.sol";
+import {console} from "forge-std/console.sol";
```

Comment:
rm

Reply:
Addressed in
- https://github.com/superstateinc/ustb/pull/56/commits/4e52d672532d51249bce9677ae1801a91404eda8
- https://github.com/superstateinc/ustb/pull/56/commits/863dda0005df872dec9136d026df07f918dbb2e5